www/javascripts/missions.js

```diff
@@ -44,7 +44,7 @@
 
 			var vm = this;
 
-			Players.getMissionStatus(this.playerId, this.missionId, function(data) {
+			Players.getMissionStatus(vm.playerId, vm.missionId, function(data) {
```

Comment:
Yep!

See: [this](https://johnpapa.net/angularjss-controller-as-and-the-vm-variable/) or [this](https://daveceddia.com/convert-scope-to-controlleras/)